sign-in/signin.css

```diff
@@ -0,0 +1,112 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+nav {
+    background-image: url(../images/concrete_wall.png);
+    color: white;
+    position: fixed;
+    width: 100%;
+}
+
+p {
+    color: white;
+    text-align: center;
+    margin-top: 40px;
+}
+
+main {
+    background-image: url(../images/concrete_wall.png);
+    min-height: 100vh;
+}
+
+.container {
+    width: 1024px;
+    max-width: 100%;
+    margin: 0 auto;
+}
+
+.brand img {
+    width: 80px;
+    height: 68px;
+    margin-top: 10px;
+}
+
+.about-me img {
+    margin-top: 80px;
+    width: 400px;
+    height: 300px;
+}
+
+.login {
+    width: 450px;
+    overflow: hidden;
+    margin: auto;
+    padding: 80px;
+    background: white;
+    height: 400px;
+    margin-top: 100px;
+}
+
+h2 {
+    text-align: center;
+    color: black;
+    padding: 20px;
+}
+
+label {
+    color: black;
+    font-size: 17px;
+}
+
+#Uname {
+    width: 300px;
+    height: 30px;
+    border: none;
+    border-radius: 15px;
+}
+
+#Pass {
+    width: 300px;
+    height: 30px;
+    border: none;
+    border-radius: 15px;
+}
+
+#log {
+    width: 300px;
+    height: 30px;
+    border: none;
+    padding-left: 7px;
+    color: white;
+}
+
+span {
+    color: black;
+    font-size: 17px;
+}
+
+a {
+    text-decoration: none;
+    color: black;
+    font-weight: 900;
+}
+
+.signin {
+    display: flex;
+    margin: auto;
+    align-items: center;
+    justify-content: center;
+}
+
+input[type="text"],
+input[type="password"] {
+    background-color: #C4C4C4;
+    border-radius: 10%;
+}
+
```

Comment:
In your inputs please add `padding 10px 20px` to have some padding. You can see how your placeholders are knocking the left input borders

![image](https://user-images.githubusercontent.com/54368821/154654762-60725382-19e3-4cdc-a0c7-75025c07c43e.png)

Reply:
Please fix those inputs with padding then we will be good to go.